allium/lib/aroi_validation.py

```
@@ -29,65 +422,255 @@ def _calc_percentage(count: int, total: int) -> float:
+    # support. If the first token is supported, use it. Otherwise log
+    # the unsupported value once (for A.8 observability) and leave
+    # version=None so the relay gets bucketed as "no AROI" until the
+    # operator fixes the contact string.
+    version = None
+    any_match = _CIISS_ANY_VERSION_RE.search(contact)
+    if any_match:
+        first_v = any_match.group(1)
+        if first_v in SUPPORTED_CIISSVERSIONS:
+            version = first_v
+        else:
+            if first_v not in _warned_unsupported_ciissversion:
```

Ruff (0.15.12)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)